CHANGELOG.md

```diff
@@ -8,13 +8,15 @@ All notable changes to this project will be documented in this file.
 
 - Add user-info-fetcher to fetch user metadata from directory services ([#433]).
 - Helm: support labels in values.yaml ([#507]).
+- Added support for OPA 0.61.0 ([#518]).
```

Comment:
Maybe we want to also mention we removed 0.51.0

Reply:
https://github.com/stackabletech/opa-operator/pull/518/commits/a3f65e779f85e416467f1b097ac1281305df94a0